src/django/pfb_analysis/filters.py

```diff
@@ -59,4 +59,9 @@ def filter_latest(self, queryset, name, value):
 
     class Meta:
         model = AnalysisJob
-        fields = ['neighborhood', 'batch', 'status', 'latest']
+        fields = {'neighborhood': ['exact', 'in'],
+                  'neighborhood__name': ['exact', 'contains'],
+                  'neighborhood__label': ['exact', 'contains'],
+                  'batch': ['exact', 'in'],
+                  'status': ['exact', 'in'],
+                  'latest': ['exact', 'in']}
```

Comment:
`'in'` doesn't work for `status`, since it's a method filter.  Adding e.g. `status__in=COMPLETE,TILING` gets quietly ignored.  Another reason to aspire to a query-based status filter.

`latest` is also a method filter, but fortunately `'in'` doesn't really make sense in that case anyway, since it's boolean.